scholly/base/src/components/App.js

```diff
@@ -19,6 +21,9 @@ const App = () => {
 
   return (
     <>
+      <Helmet>
+        <title>{WEB_PAGES.title}</title>
```

Comment:
Maybe best to follow the convention where we're assigning the global to a local variable inside the component before accessing it (to avoid any potential modifications). Could you make a local  inside this component and access `.title` from it, similar to how we're doing in other components?